src/components/ScrollViewWithPickers.js

```diff
@@ -0,0 +1,24 @@
+import React, {useRef} from 'react';
```

Comment:
I think it would be really valuable to add a comment to the top of this file to explain _why_ this component should be used as opposed to just using a normal `<ScrollView>`